common/coat_of_arms/template_lists/color_lists.txt

```diff
@@ -432,5 +524,61 @@ color_lists = {
 			125 = "yellow"
 			250 = "red_light"
 		}
+		###### Warcraft Culture Specific CoAs ######
+		
+		### Humans ###
+		special_selection = {
+			trigger = { azerothian_coa_trigger = yes }
+			30  = "white"
+			150 = "yellow"
+		}
+		special_selection = {
+			trigger = { lordaeronian_coa_trigger = yes }
+			65  = "yellow"
+			150 = "white"
+		}	
+		special_selection = {
+			trigger = { arathorian_coa_trigger = yes }
+			50  = "yellow"
```

Comment:
Don't get why it's yellow.

Reply:
[The League of Arathor](https://wowpedia.fandom.com/wiki/League_of_Arathor) uses yellow as its main colour. It goes without saying that it's not as important as Red, which is why the weight is set to 50. I can reduce it more but Arathor is another culture like Stormwind which uses only two colours: Red and White.